cli/dcoscli/util.py

```diff
@@ -54,3 +54,24 @@ def confirm(prompt, yes):
             else:
                 msg = "'{}' is not a valid response.".format(response)
                 emitter.publish(msg)
+
+def confirmText(prompt, confirmation_text):
+    """
+    :param prompt: message to display to the terminal
+    :type prompt: str
+    :param expected_text: The text to compare user input to.
+    :type expected_text: str
+    :returns: True if the user replies with the expected text; False otherwise
+    :rtype: bool
+    """
+
+    while True:
+        sys.stdout.write('{}: '.format(prompt))
+        sys.stdout.flush()
+        response = sys.stdin.readline().strip()
+        if response == confirmation_text:
+            return True
+        else:
+            msg = "Expected '{}'. You supplied '{}'.".format(confirmation_text, response)
```

Comment:
Suggestion: confirmText's  caller should probably pass outcome of failure in as text.  (maybe rename confirm_text)